Sources/bip32-ed25519-swift/CryptoUtils.swift

```diff
@@ -18,7 +18,7 @@
 import CryptoKit
 import Foundation
 
-public struct CryptoUtils {
+public enum CryptoUtils {
```

Comment:
why enum?

Reply:
The formatter changed it. I presume because CryptoUtils only have 3 functions and no internal variables keeping track of state etc?